### PR TITLE
Reo sam patch

### PR DESCRIPTION
**Pull request by reo-sam**

### CODE REVIEW

File: content/en/wiki/time-value-of-money.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Time Value of Money
```
Comment (JimWithoutJam) [inline content/en/wiki/time-value-of-money.md:8]: @reo-sam the [YAML front matter block](https://content.nuxtjs.org/writing/#front-matter) is missing :)

Here's how it looks for the DD page -
```YAML
---
title: Due Diligence (DD) checklist
category: wiki
position: 12
description: A checklist to reference while performing due diligence on a company.
---
```

**Review by JimWithoutJam — COMMENTED** [review]
Not sure if this PR is still a draft; just putting it across :)

File: content/en/wiki/time-value-of-money.md (hunk 1 above)
Comment (crimelabs786) [reply content/en/wiki/time-value-of-money.md:8]: @reo-sam share a description not exceeding 70 words, I'll add it

**Review by crimelabs786 — CHANGES_REQUESTED** [review]
share a description not exceeding 70 words, I'll add it

File: content/en/wiki/time-value-of-money.md (hunk 1 above)
Comment (reo-sam) [reply content/en/wiki/time-value-of-money.md:8]: done.

**Review by crimelabs786 — APPROVED** [review]

**Pull request merged**